docs/source/tutorial/cli/fingerprint.rst

```diff
@@ -0,0 +1,34 @@
+.. _fingerprint:
+
+Fingerprint
+============================
+
+To calculate the learned representations (encodings) of model inputs from a pretrained model, run
+
+.. code-block::
+   
+   chemprop fingerprint --test-path <test_path> --model-path <model_path> 
+
+where :code:`<test_path>` is the path to the CSV file containing SMILES strings, and :code:`<model_path>` is the location of checkpoint(s) or model file(s) to use for prediction. It can be a path to either a single pretrained model checkpoint (.ckpt) or single pretrained model file (.pt), a directory that contains these files, or a list of path(s) and directory(s). If a directory, will recursively search and predict on all found (.pt) models. By default, predictions will be saved to the same directory as the test path. If desired, a different directory can be specified by using :code:`--output <path>`
```

Comment:
```suggestion
where :code:`<test_path>` is the path to the CSV file containing SMILES strings, and :code:`<model_path>` is the location of checkpoint(s) or model file(s) to use for prediction. It can be a path to either a single pretrained model checkpoint (.ckpt) or single pretrained model file (.pt), a directory that contains these files, or a list of path(s) and directory(s). If a directory, will recursively search and predict on all found (.pt) models. By default, predictions will be saved to the same directory as the test path. If desired, a different directory can be specified by using :code:`--output <path>`. The output <path> can end with either .csv or .npz, and the output will be saved to the corresponding file type.
```

Reply:
thanks. Is this change true also for `predict` (`preds-path`)?